### PR TITLE
[FLINK-13677][docs-zh] Translate "Monitoring Back Pressure" page into Chinese

### PR DESCRIPTION
**Pull request by 1996fanrui**
## What is the purpose of the change

Translate "Monitoring Back Pressure" page into Chinese

## Brief change log

Translate "Monitoring Back Pressure" page into Chinese

## Verifying this change

This change is to add a new translated document.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no

### CODE REVIEW

**Comment by 1996fanrui** [comment]
@wuchong Can you take a look when you are free?

**Comment by 1996fanrui** [comment]
@wuchong Hi, can you review this pr? thanks

**Comment by wuchong** [comment]
Yes. I will review it. Sorry for the late response.

File: docs/monitoring/back_pressure.zh.md [hunk 1]
```diff
@@ -22,59 +22,62 @@ specific language governing permissions and limitations
...
-- `web.backpressure.refresh-interval`: Time after which available stats are deprecated and need to be refreshed (DEFAULT: 60000, 1 min).
-- `web.backpressure.num-samples`: Number of stack trace samples to take to determine back pressure (DEFAULT: 100).
-- `web.backpressure.delay-between-samples`: Delay between stack trace samples to determine back pressure (DEFAULT: 50, 50 ms).
+- `web.backpressure.refresh-interval`: 有效统计被废弃并重新进行采样的时间 (默认: 60000, 1 min)。
+- `web.backpressure.num-samples`: 用于确定反压的 stack trace 样本数 (默认: 100)。
+- `web.backpressure.delay-between-samples`: 用于确定反压的 stack trace 采样间隔时间 (默认: 50, 50 ms)。
 
 
-## Example
+## 案例
```
Comment (wuchong) [inline docs/monitoring/back_pressure.zh.md]:
```suggestion
## 示例
```

File: docs/monitoring/back_pressure.zh.md [hunk 2]
```diff
@@ -22,59 +22,62 @@ specific language governing permissions and limitations
...
 
 * ToC
 {:toc}
 
-## Back Pressure
+## 反压
 
-If you see a **back pressure warning** (e.g. `High`) for a task, this means that it is producing data faster than the downstream operators can consume. Records in your job flow downstream (e.g. from sources to sinks) and back pressure is propagated in the opposite direction, up the stream.
+如果你看到一个 Task 发生 **反压警告**（例如： `High`），意味着它生产数据的速率比下游 Task 消费数据的速率要快。
+在工作流中数据记录是从上游向下游流动的（例如：从 Sources 到 Sinks）。反压沿着相反的方向传播，沿着数据流向上游传播。
```
Comment (wuchong) [inline docs/monitoring/back_pressure.zh.md]:
```suggestion
在工作流中数据记录是从上游向下游流动的（例如：从 Source 到 Sink）。反压沿着相反的方向传播，沿着数据流向上游传播。
```

**Comment by 1996fanrui** [comment]
@wuchong Thank you very much

**Pull request closed without merging**